VCFv4.3.tex

```diff
@@ -503,41 +503,100 @@ \subsection{VCF tag naming conventions}
 
 \section{INFO keys used for structural variants}
```

Comment:
@d-cameron ,
> Overall, this change weakens the specifications. The VCF specifications should be a common file format in which multiple programs can unambiguous interpret and parse variants. As it stands these changes would make it no longer possible to unambiguously convert a VCF SV record to another format such as BEDPE.

Please help me understand your point. To which changes are you referring? Please elaborate, perhaps including an example that helps clarify your objection.